sample_data_config/ci-ejp-xml-data-pipeline.config.yaml

```diff
@@ -0,0 +1,12 @@
+gcpProjectName: 'elife-data-pipeline'
+dataPipelineId: ejp_xml_pipeline_id
+dataset: '{ENV}'
+manuscriptTable: 'manuscript_truncated_{ENV}'
```

Comment:
my suggestion would be to just add a short comment to the file, mentioning that the tables in here are getting truncated during the end2end test. (maybe I would use `temp` instead of `truncated` but that may just be the first thing that came into my biased head)